gateway/ln-gateway/src/lightning/lnd.rs

```diff
@@ -1244,7 +1244,8 @@ impl ILnRpcClient for GatewayLndClient {
                         };
 
                     ChannelInfo {
-                        remote_pubkey: channel.remote_pubkey,
+                        remote_pubkey: PublicKey::from_str(&channel.remote_pubkey)
+                            .expect("Server returned invalid remote channel pubkey"),
```

Comment:
Server means lightning node in this context, right?

Reply:
Yep! I fixed the message to make that more clear.